client/app/home/home.html

```diff
@@ -8,6 +8,32 @@ <h1 class="section__title">
            href="/contribute"><span>{{ 'Start Mapping' | translate }}</span></a>
     </div>
 </header>
+<section class="section__body--secondary--dark">
+    <div class="homepage-stats-container row--centered">
+        <div class="inner">
+            <ul class="homepage-stats__cards">
+                <li id="homepage-stats-1" class="homepage-stats__card">
+                    <div class="stats-card__amount">
+                        <strong>{{ homeCtrl.mappersOnline | number }}</strong>
+                    </div>
+                    <div class="stats-card__title">Mappers Online</div>
```

Comment:
Could we make this and the other two card titles translatable?

Reply:
@ethan-nelson I think this work would be better done as a seperate PR.

I did take took at how I might generate the transifex keys but there's no good docs on the process.  @LindaAlblas @bgirardot can you describe the process for generating the transifex keys?

Reply:
To make it translatable, the text would be `{{ 'Mappers Online' | translate }}` instead of `Mappers Online`. But yes we can definitely do that in another PR.